examples/next/pages/ui/components/account-settings/change-password/aws-exports.js

```diff
@@ -1,2 +1,2 @@
-import awsExports from '@environments/auth/auth-with-optional-totp-and-sms-mfa/src/aws-exports';
+import awsExports from '@environments/auth/auth-with-email/src/aws-exports';
```

Comment:
`auth-with-optional-totp-and-sms-mfa` seems to not be set up correctly. Also, for consistency, since delete-user app uses auth-with-email.